public/content/posts/performance-improvement-in-a-next.js-page/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Performance improvements in a Next.js page"
+date: "2022-09-12T09:30:08.625Z"
+subtitle: "When improving the performance, web vitals take a vital place. In this article we are mainly looking at the LCP score."
+imageSrc: performance.png
+imageAlt: Performance score in lighthouse
+author: gayan-witharana
+tags:
+  - performance
+  - web-development
+  - Next.js
+  - web-vitals
+---
+
+## Are web vitals necessary?
+
+From Google monthly user experience data, between May and October 2021, overall Core web vitals compliance increased by 58 per cent worldwide.
+
+- Largest Contentful Paint – 44 per cent compliance
+- Cumulative Layout Shift – 71 per cent met requirements
+- First Input Delay – 91 per cent compliance
+
+Reference: https://www.figmentagency.com/seo/core-web-vitals-seo-updates-for-2022
+
+So obviously if you're not optimizing for core web vitals, you are definitely in the minority.
+
+## LCP
+
+In the Fishbrain web application, we're doing fine with the FID and CLS scores as you can see in the title image. But the LCP score is quite hight compared to the standard time (2.5s).
+
+![LCP score margin](/lcp.png "LCP Score")
+
+### Identifying the LCP of a page
+
+There are few ways we can identify the LCP of a page. One easy way is to use the Performance Insights tab in Google Chrome. Once you click the "Measure page load" button it will run for a few seconds and then it will show some details related to performance. On the right side of the tab, you will be able to see web vitals and once you hover LCP it will highlight the LCP content of the page.
+
+![Google Chrome Performance Insights](/perf-insight.png "Google Chrome Performance Insights")
+
+### Improve the performance of an image
+
+Once we identify the LCP element of the page, we can try to make sure it loads relatively fast. Since in this case it's an image we can load the image in "eager" mode instead of the "lazy" mode.
```

Comment:
```suggestion
Once we identify the LCP element of the page, we can try to make sure it loads relatively fast. Since in this case it's an image, we can load the image in "eager" mode instead of "lazy" mode.
```